index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Module 1 capstone project</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+  <script src='main.js'></script>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+<body>
+  <!-- Navbar --> 
+  <input id="toggle" type="checkbox"></input>
+
+  <label for="toggle" class="hamburger container-fluid px-4">
+    <div class="top-bun border"></div>
+    <div class="meat border"></div>
+    <div class="bottom-bun border"></div>
+  </label>
+
+  <div class="nav">
+    <div class="nav-wrapper">
+      <nav>
+        <a href="#home">HOME</a><br>
+        <a href="#main-program">PROGRAM</a><br>
+        <a href="#featured-speakers">SPEAKERS</a><br>
+        <a href="#join">JOIN</a>
+      </nav>
```

Comment:
Looking at the template `Desktop Size` there are two navbars

- The nav is divided into a top dark bar and a bottom white bar.
- The top short bar section with social media links, log-out, and language links. 
- The taller bottom bar with a logo and a list of page links.

Template
![image](https://user-images.githubusercontent.com/78251927/233807492-67b74e47-e7a7-4955-ac97-2756fd688dc1.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/233807509-6e930fea-d80f-4b76-987f-5f828b7c29a7.png)

Please kindly add the missing `top navbar` 🛠💪